src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -472,6 +472,7 @@
 
         :else
         {:status 200
+         :headers {"Content-Type" "application/octet-stream"}
```

Comment:
Yeah, I think adding `text/plain` would be reasonable to do for the error responses - the 400 and 403s response maps above and would be good with adding that to this PR while we're in here.

> Should we be setting a default content-type of text/plain if no content-type was given?

@finch For this one, do you mean for the success case or just the fail cases?  I had been thinking for the success case that we'd always return `application/octet-stream`.